include/glow/Runtime/RuntimeTypes.h

```diff
@@ -81,6 +81,15 @@ struct DAGNodeList {
   std::vector<std::unique_ptr<DAGNode>> nodes;
 };
 
+/// This is the base struct for DeviceManager configurations. The base struct
+/// contains a string for the device name. Any specific device can extend this
+/// struct to contain additional information to identify and configure the
+/// device manager.
+struct DeviceConfig {
+  /// A string containing the name of the device the device manager controls.
+  std::string deviceName;
```

Comment:
You have deviceName as a parameter here but you've deleted every point in the code that set a device name as far as I can tell. Do we need this?

Reply:
Probably not, I'll go ahead and remove it.